appinfo/routes.php

```diff
@@ -320,6 +320,11 @@
 			'url' => '/api/settings/allownewaccounts',
 			'verb' => 'POST'
 		],
+		[
+			'name' => 'settings#setEnabledThreadSummary',
+			'url' => '/api/settings/threadsummary',
+			'verb' => 'POST'
```

Comment:
can be a PUT because the operation is idempotent. It would be ok for the frontend to send the same requests more than once with the same result.